### ISSUE TITLE
aria-live="off"

### ISSUE THREAD

**Issue by kengdoj** [Section508Coordinators/ICTTestingBaseline#176]
If aria-live="off", is an AT user being notified at all that a change has occurred? Would it pass notification of change (4.1.2) tests?

**Comment by kengdoj** [comment]
Related to the following test(s), where aria live region is mentioned:

- [ ] 5. Changing Content

I don't think "off" should be an acceptable politeness_setting for aria-live, if the live region is being used to notify user of a change of content.

From https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions:
> For regions which are not important, or would be annoying because of rapid updates or other reasons, silence them with aria-live="off".

From https://bitsofco.de/using-aria-live/:

> aria-live="off"
The off value is practically the same as if the attribute was omitted entirely. Assistive technology will not keep track of changes to the elements content and, as such, will not notify its users of any changes.

**Comment by drewnielson** [comment]
I'm not sure I agree that including setting the politeness to "off" is the same as not having a live region at all. I will need to investigate further, but I believe that some assistive technologies still provide the ability to query for live regions even when politeness is set to off. So there is still some benefit to defining a live region even if the politeness is set to off.

**Comment by drewnielson** [comment]
So, I'm definitely wrong, at least for NVDA. It does not provide any reference to live regions in the user guide or quick access keyboard shortcuts reference. ANDI reports default politeness setting, so I think we should be good to instruct testers to check for aria-live="off" and give a fail if so.

**Comment by kengdoj** [comment]
1. Identify if any roles that have default politeness setting as off. If none, check live regions used for change notification are not set to aria-live="off". Do not check if any other politeness setting is appropriate.
2. Alan will create test cases for "off" and user configurable politeness settings for review.

**Comment by kengdoj** [comment]
In Conforming Alternate Version test, if there is an option to enable aria-live regions, enable. 

In Changing Content test, if page provides option to enable aria-live regions, enable.
